http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/KubernetesServiceDbQueries.scala

```diff
@@ -1,26 +1,33 @@
 package org.broadinstitute.dsde.workbench.leonardo
 package db
 
-import java.time.Instant
 import akka.http.scaladsl.model.StatusCodes
 import cats.data.Chain
 import cats.syntax.all._
+import org.broadinstitute.dsde.workbench.google2.KubernetesClusterNotFoundException
+import org.broadinstitute.dsde.workbench.leonardo.db.DBIOInstances._
 import org.broadinstitute.dsde.workbench.leonardo.db.LeoProfile.api._
 import org.broadinstitute.dsde.workbench.leonardo.db.LeoProfile.mappedColumnImplicits._
+import org.broadinstitute.dsde.workbench.leonardo.db.appQuery.{filterByWorkspaceId, nonDeletedAppQuery}
 import org.broadinstitute.dsde.workbench.leonardo.db.kubernetesClusterQuery.unmarshalKubernetesCluster
 import org.broadinstitute.dsde.workbench.leonardo.db.nodepoolQuery.unmarshalNodepool
 import org.broadinstitute.dsde.workbench.leonardo.http.GetAppResult
 import org.broadinstitute.dsde.workbench.leonardo.model.LeoException
-import DBIOInstances._
-import org.broadinstitute.dsde.workbench.google2.KubernetesClusterNotFoundException
-import org.broadinstitute.dsde.workbench.leonardo.db.appQuery.nonDeletedAppQuery
 import org.broadinstitute.dsde.workbench.model.TraceId
 
+import java.time.Instant
 import scala.concurrent.ExecutionContext
 
 object KubernetesServiceDbQueries {
-  // the 'full' here means that all joins possible are done, meaning all fields in the cluster, nodepool, and app will be present
-  // if you just need the cluster, nodepools, and cluster-wide namespaces, see the minimal join in the KubernetesClusterComponent
+
+  /**
```

Comment:
I tried to comment the public methods here, and make clear which should be used for v1 apps and v2 apps.

In general v2 apps should query based on `workspace`, and v1 should query based on `cloudContext`.